### ISSUE TITLE
After upgrading to Bullet 7.1, my Rails 6.1 tests start failing (reset_counters)

### ISSUE THREAD

**Issue by mikdiet** [flyerhzm/bullet#682]
I'm trying to update from 7.0.7 to 7.1.0, and getting this:

```
      ArgumentError:
        wrong number of arguments (given 1, expected 0)
      # /usr/local/bundle/gems/bullet-7.1.0/lib/bullet/active_record61.rb:275:in `count'
```

The caller does `reset_counters` on one of my models.

**Comment by DeveloperSensorberg** [comment]
Reason: `count` takes an argument: https://api.rubyonrails.org/v7.0.8/classes/ActiveRecord/Calculations.html#method-i-count

Root cause: #655 added overrides for count which don't take an argument.

**Comment by flyerhzm** [comment]
@mikdiet I have released version 7.1.1 to fix this issue.
